client/pages/car-details.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Card, Table } from 'react-bootstrap';
+
+class CarDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      carTitle: '',
+      carPhoto: '',
+      carRecords: []
+    };
+    this.makeTable = this.makeTable.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/garage/details/${this.props.vehicleId}`)
+      .then(result => result.json())
+      .then(result => {
+        const { year, make, model, photoUrl } = result[0];
+        this.setState({
+          carTitle: `${year} ${make} ${model}`,
+          carPhoto: photoUrl
+        });
+      });
+    fetch(`/api/garage/recent-history/${this.props.vehicleId}`)
+      .then(result => result.json())
+      .then(result => {
+        this.setState({
+          carRecords: result[0].records
+        });
+      })
+      .catch(err => console.error(err));
```

Comment:
I could be wrong, but I think these are redundant. You should be able to just use one route that gives you the car's details as well as the recent records. (The query we talked about)

Reply:
Removed lines 16-24
![image](https://user-images.githubusercontent.com/48267398/163466612-6a75fd31-c69b-4730-9119-d7918c16f1fd.png)